printrun/gl/libtatlin/actors.py

```diff
@@ -122,9 +122,9 @@ def draw(self):
         glTranslatef(self.xoffset, self.yoffset, self.zoffset)
 
         def color(i):
-            if i % self.graduations_major == 0:
+            if i % self.grid[1] == 0:
                 glColor4f(*self.color_grads_major)
-            elif i % (self.graduations_major // 2) == 0:
+            elif i % (self.grid[1] // 2) == 0:
                 glColor4f(*self.color_grads_interm)
```

Comment:
The above line is never reached. It would make every other major tick more transparent.
@kliment Do we want it fixed or removed? It's not from this PR.

Reply:
I would instinctively say remove.